Translations/pt_BR.po

```diff
@@ -1568,128 +1559,132 @@ msgstr "Pausa"
 
 #: Source/help.cpp:26
 msgid "$Keyboard Shortcuts:"
-msgstr ""
+msgstr "$Atalhos de teclado:"
 
 #: Source/help.cpp:27
 msgid "F1:    Open Help Screen"
-msgstr ""
+msgstr "F1:    Abrir tela de ajuda"
 
 #: Source/help.cpp:28
 msgid "Esc:   Display Main Menu"
-msgstr ""
+msgstr "Esc:   Mostrar menu principal"
 
 #: Source/help.cpp:29
 msgid "Tab:   Display Auto-map"
-msgstr ""
+msgstr "Tab:   Mostrar automapa"
 
 #: Source/help.cpp:30
 msgid "Space: Hide all info screens"
-msgstr ""
+msgstr "Espaço: Ocultar todas as janelas de informação"
```

Comment:
"janelas" (windows) is a bit different here. A literal (and acceptable) translation would be:
```suggestion
msgstr "Espaço: Ocultar todas as telas de informação"
```
Although I'd suggest we change the original to "info panels" and use "todos os paineis" in portuguese.